packages/loader/src/gltf/parser/GLTFParserContext.ts

```diff
@@ -36,6 +36,9 @@ export class GLTFParserContext {
 
   private _resourceCache = new Map<string, any>();
 
+  /** @internal */
+  _bufferViewMap = new WeakMap<IBufferView, Promise<Uint8Array>>();
```

Comment:
GLTFParserContext will dispose when parse finished

Reply:
`GLTFParserContext` will cache automatically in `_resourceCache`

Reply:
done